plugins/jenkins-backend/src/service/router.ts

```diff
@@ -131,17 +131,18 @@ export async function createRouter(
     '/v1/entity/:namespace/:kind/:name/job/:jobFullName/:buildNumber::rebuild',
     async (request, response) => {
       const { namespace, kind, name, jobFullName } = request.params;
+      const token = getBearerTokenFromAuthorizationHeader(
+        request.header('authorization'),
+      );
       const jenkinsInfo = await jenkinsInfoProvider.getInstance({
         entityRef: {
           kind,
           namespace,
           name,
         },
         jobFullName,
+        backstageToken: token,
```

Comment:
Unrelated to this change but it's a bit unfortunate with the mixed names. It's called `backstageToken` here and only passed as `token` further down

Reply:
@jhaals do you suggest to rename `token` to backstageToken? There was a discussion in another PR to define the backstage's token explicitly. Because Jenkins itself uses also a token for API calls.

Reply:
I think it's alright as is, matches the implementation of the other endpoints